### PR TITLE
Fix compliance test

### PR DESCRIPTION
**Pull request by fbouliane**
This enables the compliance test for get_interface and get_vlan_interfaces to make sure everything is compliant.

### CODE REVIEW

File: netman/adapters/switches/dell.py [hunk 1]
```diff
@@ -343,41 +378,10 @@ def parse_vlan_list(result):
...
-            for i in range(int(start), int(end)+1):
-                interface_list.append("ethernet {0}{1}".format(debut, i))
-        elif regex.match('^(\d/[a-z]+)(\d+)', port):
-            debut, start = regex
-            interface_list.append("ethernet {0}{1}".format(debut, start))
-    return interface_list
+
+
+
+
```
Comment (JoProvost) [inline netman/adapters/switches/dell.py]: Empty lines

File: netman/adapters/switches/brocade.py [hunk 2]
```diff
@@ -308,7 +308,7 @@ def get_vlan_interfaces(self, vlan_number):
             if regex.match("(Untagged|Statically tagged) Ports\s+: (.*)$", line):
                 for real_name in _to_real_names(parse_if_ranges(regex[1])):
                     interfaces.append(real_name)
-        return interfaces
+        return sorted(interfaces)
```
Comment (JoProvost) [inline netman/adapters/switches/brocade.py]: Why do we need to sort here?

Will it output ["ethernet 1", "ethernet 2", "ethernet 3"... "ethernet 10", "ethernet 11"] or ["ethernet 1", "ethernet 10", "ethernet 11", "ethernet 2", "ethernet 3" ...]

Comment (fbouliane) [inline netman/adapters/switches/brocade.py]: ethernet 1, ethernet 10, unverified. I will compare a set in the test instead.

**Comment by fbouliane** [comment]
Comments adressed, Back to green  :white_check_mark:

File: tests/adapters/compliance_tests/get_vlan_interfaces_test.py [hunk 3]
```diff
@@ -30,22 +27,27 @@ def setUp(self):
...
-                    is_([self.test_ports[0].name, self.test_ports[1].name, self.test_ports[2].name]))
+                    contains_inanyorder(self.test_ports[0].name, self.test_ports[1].name))
 
     def test_fails_when_the_vlan_does_not_exist(self):
         with self.assertRaises(UnknownVlan):
             self.client.get_vlan_interfaces(2000)
 
     def tearDown(self):
+        self.janitor.unset_interface_access_vlan(self.test_ports[0].name)
+        self.janitor.unset_interface_native_vlan(self.test_ports[0].name)
```
Comment (lindycoder) [inline tests/adapters/compliance_tests/get_vlan_interfaces_test.py]: the test says we don't touch native vlan, wierd to remove it

Comment (fbouliane) [inline tests/adapters/compliance_tests/get_vlan_interfaces_test.py]: I agree, was a mistake, now fixed.

**Comment by lindycoder** [comment]
:+1:

**Comment by JoProvost** [comment]
:+1:

**Pull request merged**